agentMET4FOF/agents.py

```diff
@@ -456,18 +456,30 @@ def send_plot(self, fig=plt.Figure()):
         fig : Figure
             Can be either matplotlib figure or plotly figure
 
+        mode : str
+            "image" - converts into image via encoding at base64 string.
+            "plotly" - converts into plotly figure using `mpl_to_plotly`
+            Default: "image"
+
         Returns
         -------
         The message format is {'from':agent_name, 'plot': data, 'senderType': agent_class}.
         """
+
```

Comment:
I would suggest doing a quick check for the two input parameters like:  is fig of either of the two expected types or a dict of those and is mode one of "image" or "plotly". Otherwise we should do theses checks as `else` clauses in the conditions later on, which would be less clear and more error prone.